16028 settings and more context menu not accessible in ie10

hschink opened the pull request:
[`* `#16028`Context menus in IE10 not accessible`](https://www.openproject.org/work_packages/16028)

This is a follow up branch of #2001 and fixes the tabbing for the settings and more menu (see [comment in the related ticket](https://community.openproject.org/work_packages/16028#note-16)).

Commit:
Add empty href

Empty hrefs allow IE10 to properly focus the elements.

TeatroIO commented:
I've prepared a stage. <a href='http://fix-16028-fix-settigs-and-more-menu.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

mfrister commented:
Looks great :smiley: 

I'd like something that helps a future reader of the code understand why the `a` tags have a `href=""` attribute. A comment at the first usage of these attributes in each file would be my idea.

Commit:
Add comment to hrefs with empty URLs

This should prevent fellow developers from accidentally removing the previous fix for IE 10.

hschink commented:
@meeee Thanks for the hint! :+1: The missing comments should be fixed with https://github.com/opf/openproject/commit/b1b7f2b8f7d914f5dfbfe8e1f7b2db632d818baf.

mfrister commented:
:+1:
